roles/kubernetes-apps/cluster_roles/tasks/main.yml

```diff
@@ -3,8 +3,6 @@
   uri:
     url: "{{ kube_apiserver_endpoint }}/healthz"
     validate_certs: no
-    client_cert: "{{ kube_apiserver_client_cert }}"
```

Comment:
This should not be removed, when using your external CA you should override the values of these variables if needed.